src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/antlr4/ExpectedAntlr4AstRegressionCaseDefault.txt

```diff
@@ -71,14 +72,15 @@ COMPILATION_UNIT -> COMPILATION_UNIT [2:0]
         |       |   |   |   |       |   |   |--TYPE -> TYPE [10:18]
         |       |   |   |   |       |   |   |   `--IDENT -> String [10:18]
         |       |   |   |   |       |   |   `--IDENT -> s [10:25]
-        |       |   |   |   |       |   `--GT -> > [10:41]
-        |       |   |   |   |       |       |--METHOD_CALL -> ( [10:38]
-        |       |   |   |   |       |       |   |--DOT -> . [10:31]
-        |       |   |   |   |       |       |   |   |--IDENT -> s [10:30]
-        |       |   |   |   |       |       |   |   `--IDENT -> length [10:32]
-        |       |   |   |   |       |       |   |--ELIST -> ELIST [10:39]
-        |       |   |   |   |       |       |   `--RPAREN -> ) [10:39]
-        |       |   |   |   |       |       `--NUM_INT -> 6 [10:43]
+        |       |   |   |   |       |   `--EXPR -> EXPR [10:41]
```

Comment:
All diffs in AST files are expected, we have just added a new `EXPR` node parent to each.